distribution/server-x-dist/src/main/content/themes/README.md

```diff
@@ -0,0 +1,22 @@
+# Creating Themes
```

Comment:
Should this really be a markdown file? Usually this file is just opened in a text editor, and not many folks have a markdown viewer installed

Reply:
Yeah. I changed because the same file in the `providers` dir is an MD. Do you want to change to text too?